pkgs/top-level/all-packages.nix

```diff
@@ -20675,7 +20675,7 @@ in
 
     digimend = callPackage ../os-specific/linux/digimend { };
 
-    dpdk = callPackage ../os-specific/linux/dpdk { };
+    dpdk-kmods = callPackage ../os-specific/linux/dpdk-kmods { };
```

Comment:
Please add an alias entry and remove the old file.

Reply:
We can also add a throw alias which just informs the user about this change.

Reply:
Ok. Added. How do we now it is not breaking anything?

Reply:
Aberrantly it broke something.